### ISSUE TITLE
Updating an unlinked token actor while its token document configuration is open can cause an update for the token document to also be fired

### ISSUE THREAD

**Issue by Fyorl** [foundryvtt/foundryvtt#8564]
Originally reported by Wasp#2005 https://discord.com/channels/170995199584108546/956306859491471420/1046051399496122398

**ALL MODULES DISABLED?** true

**OS, Hosting, Browser (if applicable):**  
Windows, NodeJS, Chrome (latest version)

**Short Description of bug:**  
Updating an unlinked token's configuration through the token config UI causes the actor to be updated

**Simple steps to reproduce the bug:**
1. Create unlinked actor in a fresh world
2. Drag token onto empty scene
3. Run the following in the console:
```js
Hooks.on("updateActor", (...args) => { console.log("ACTOR", args) });
Hooks.on("updateToken", (...args) => { console.log("TOKEN", args) });
```

4. Double right click on the token and update its rotation to 90
5. Only `updateToken` is fired
6. Run the following in the console with the token selected:
```js
_token.actor.setFlag("world", "test", true)
```

7. Observe both the token and actor hooks are fired, with bogus data on the token changes
8. Update the token's rotation via the token configuration UI to 0
9. Observe both the token and actor hooks are fired again, with bogus data on the actor changes

![image](https://user-images.githubusercontent.com/207433/204285102-21a0a58f-1fc9-4d67-9816-663d65bb9e78.png)

**Comment by Fyorl** [comment]
I suspect this is the same root cause as #8368 and likely already fixed in the upcoming release.

**Comment by Fyorl** [comment]
Confirmed that this was fixed by the changes to address #8368 .
